src/hapi/pipelines/configs/population.yaml

```diff
@@ -1,6 +1,24 @@
 # Collector specific configuration
 commit_limit: 1000
 
+default:
+  keys:
+    - "scraper_national"
+    - "scraper_adminone"
+    - "scraper_admintwo"
+  input:
+    - "F_TL"
+    - "M_TL"
+    - "T_TL"
+  output:
+    - "F_TL"
+    - "M_TL"
+    - "T_TL"
+  output_hxl:
+    - "#population+f+total"
+    - "#population+m+total"
+    - "#population+total"
```

Comment:
Could we add to the default what I've highlighted in yellow [here](https://docs.google.com/spreadsheets/d/1JM-NxFJSkursHZOrBiupmRJesujhVtSyC8ts0qwoz3c/edit#gid=910577744)? (But also include m, f, and t).